src/Middleware/SpaServices.Extensions/src/SpaOptions.cs

```diff
@@ -69,6 +70,27 @@ public PathString DefaultPage
         /// </summary>
         public string SourcePath { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name of the package manager executible, (e.g npm,
+        /// yarn) to run the SPA.
+        /// 
+        /// If not set, npm will be assumed as the default package manager 
+        /// executable
```

Comment:
End this sentence with a period.

Reply:
I would prefer instead of this second sentence to say `The default value is 'npm'.` It's not so much about not being set, but about the default value.

Reply:
Fixed